fix: code block with incorrect language

sumimakito opened the pull request:
According to the context, the below code block should be for a JSON file. However, it is rendered as `cypress.config.js/ts`:

<img width="1139" alt="Screenshot 2024-08-16 at 14 44 28" src="https://github.com/user-attachments/assets/0b69dbe2-da66-4694-b2a3-79544d5adcb5">

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/cypress-io/cypress-documentation?pullRequest=5896) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/cypress-io/cypress-documentation?pullRequest=5896) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/cypress-io/cypress-documentation?pullRequest=5896) it.</sub>

MikeMcC399 commented:
@sumimakito 

There is special pre-processing used on this page and your change fails formatting checks, so I suspect that it may not be necessary to change. You could check how `:::cypress-config-example` is supposed to work.

sumimakito commented:
Hi @MikeMcC399, thanks for responding. It turned out that it was caused by the incorrect format in the JSON code block (unquoted keys and single quotes around values). I've fixed the issue and updated the PR.

From [the documentation](https://github.com/cypress-io/cypress-documentation/blob/04b99f007c25f100811583705970a2bb5ec5cd49/plugins/cypressRemarkPlugins/src/cypressConfigExample/readme.md), `:::cypress-config-example` directive wraps a JavaScript object with `defineConfig(...)` and marks it as a default export, but in this PR, I was intended to render the JSON configuration with a proper JSON code block instead of a JavaScript/TypeScript block rendered by the `:::cypress-config-example` directive.

MikeMcC399 commented:
@sumimakito 

I understand now that you are correcting an issue in https://docs.cypress.io/guides/tooling/reporters#Command-line-3 which is supposed to display an example `reporter-config.json` and this example was wrongly generated / formatted. Sorry for my misunderstanding!

sumimakito commented:
@MikeMcC399 No problem. Thanks for helping check the CI results anyway!

jennifer-shehane reviewed (approved):
@sumimakito Thanks for the fix!
